src/Azure.Functions.Cli/Diagnostics/ColoredConsoleLogger.cs

```diff
@@ -55,12 +55,7 @@ private void LogToConsole(LogLevel logLevel, Exception exception, string formatt
         {
             foreach (var line in GetMessageString(logLevel, formattedMessage, exception))
             {
-                var outputline = $"{line}";
-                if (_loggingFilterHelper.VerboseLogging)
-                {
-                    outputline = $"[{DateTime.UtcNow}] {outputline}";
-                }
-                ColoredConsole.WriteLine($"{outputline}");
+                ColoredConsole.WriteLine($"[{DateTime.UtcNow}] {line}");
```

Comment:
We typically format this as `DateTime.UtcNow.ToString("O")` -- or even shorten the milliseconds down a bit sometimes with `ToString("yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffK")`.

Reply:
fixed!